test/functional/p2p_outbound_eviction.py

```diff
@@ -207,12 +207,40 @@ def test_outbound_eviction_mixed(self):
         for peer in misbehaving_unprotected_peers:
             peer.wait_for_disconnect()
 
+        node.disconnect_p2ps()
```

Comment:
Annoying nit: This should be in the previous commit (feel free to disregard)

Reply:
I don't think that's the case. In the previous commit this was the last test so there was no need to manually disconnect the nodes. This is only needed after the last commit to start fresh

Reply:
On second thought it seems a bad idea to depend on the order in which tests are called for deciding whether or not to perform cleanup. I think that might be kind of a footgun for anyone modifying the test in the future

Reply:
Ok, agreed